### PR TITLE
Zmtp3

### PR DESCRIPTION
**Pull request by somdoron**

### CODE REVIEW

**Comment by somdoron** [comment]
@drewnoakes this is ready to be reviewed and merged

File: src/NetMQ/Core/Utils/ByteArrayUtility.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+namespace NetMQ.Core.Utils
+{
+    public static class ByteArrayUtility
```
Comment (drewnoakes) [inline src/NetMQ/Core/Utils/ByteArrayUtility.cs]: Potentially make this `internal`.

Comment (somdoron) [reply src/NetMQ/Core/Utils/ByteArrayUtility.cs]: fixed

**Comment by somdoron** [comment]
Thanks, this is actually a preparation for the curve encryption.

**Pull request merged**